### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by nemchik**
Logo PR: https://github.com/linuxserver/docker-templates/pull/232

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+github: linuxserver
+open_collective: linuxserver
+custom: ["https://patreon.com/overseerr",]
```
Comment (Roxedus) [inline .github/FUNDING.yml:3]: Maybe use the github sponsor link, as it has less fees?

Comment (nemchik) [reply .github/FUNDING.yml:3]: I'll add the gh sponsor link as well so it'll show both.

**Comment by nemchik** [comment]
This should be good to go now.

File: readme-vars.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+common_param_env_vars_enabled: true
+param_container_name: "{{ project_name }}"
+
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "TZ", env_value: "America/New_York", desc: "Specify a timezone to use EG America/New_York"}
+
+param_usage_include_ports: true
+param_ports:
+  - { external_port: "5055", internal_port: "5000", port_desc: "Port for Overseerr's web interface." }
```
Comment (aptalca) [inline readme-vars.yml:34]: shouldn't the internal be 5055?

Comment (nemchik) [reply readme-vars.yml]: Resolved in abb8948eb611dfc1654218f6cbd6b3691f53ff91

**Review by aptalca — APPROVED** [review]

**Pull request merged**